### DISCUSSION TITLE
List of JSON as Input without using BaseModel

### DISCUSSION THREAD

**Discussion by alivcor** [fastapi/fastapi#7760] [Questions] [answered]
### First check

* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.

### Description

How can I have a generic JSON as an input

```
"params": {
    "varA": {
      "x": "2",
      "version": 4
    },
    "y":  [{
              "p": 228.0,
              "q": "abcd"
            }]
}
```
If I define `y: List[Json]` I get:

"msg": "JSON object must be str, bytes or bytearray",
`HTTP/1.1" 422 Unprocessable Entity`

or `y: List[str]` or `y: str` 
`HTTP/1.1" 400 Bad Request`

Is it possible to have a generic JSON List as an input instead of having to define a hard-coded Pydantic BaseModel?

**Comment by phy25** [comment, marked as answer]
`y: List[dict]` or `y: List[typing.Any]`?

**Comment by alivcor** [comment]
@phy25 - Thanks

**Comment by tiangolo** [comment]
Thanks for the help here @phy25 ! :cake: :bowing_man: 

And thanks @alivcor for reporting back and closing the issue :+1:
